packages/mdc-radio/README.md

```diff
@@ -46,176 +46,89 @@ npm install @material/radio
+`mdc-radio-checked-stroke-color($color)` | Sets the stroke color of a checked radio button
+`mdc-radio-ink-color($color)` | Sets the ink color of a radio button
+`mdc-radio-focus-indicator-color($color)` | Sets the color of the focus indicator
 
-Boolean. Proxies to the foundation's `isChecked`/`setChecked` methods when retrieved/set
-respectively.
+#### Caveat: Edge and CSS Custom Properties
 
-##### MDCRadio.disabled
+In browsers that fully support CSS custom properties, the above mixins will work if you pass in a [MDC Theme](mdc-theme) property (e.g. `primary`) as an argument. However, Edge does not fully support CSS custom properties. If you are using any of the Sass mixins, you must pass in an actual color value for support in Edge.
 
-Boolean. Proxies to the foundation's `isDisabled/setDisabled` methods when retrieved/set
-respectively.
+## `MDCRadio` Properties and Methods
 
-##### MDCRadio.value
+Property | Value Type | Description
+--- | --- | ---
+`checked` | Boolean | Proxies to the foundation's `isChecked`/`setChecked` methods
+`disabled` | Boolean | Proxies to the foundation's `isDisabled/setDisabled` methods
+`value` | String | Proxies to the foundation's `getValue/setValue` methods
 
-String. Proxies to the foundation's `getValue/setValue` methods when retrieved/set
-respectively.
+## Usage within Web Frameworks
 
-### Using the Foundation Class
+If you are using a JavaScript framework, such as React or Angular, you can create a Radio button for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](../../docs/integrating-into-frameworks.md).
 
-Since MDC Radio is primarily driven by its native control, the adapter API is extremely simple.
+### `MDCRadioAdapter`
 
 | Method Signature | Description |
 | --- | --- |
-| `getNativeControl() => HTMLInputElement?` | Returns the native radio control, if available. Note that if this control is not available, the methods that rely on it will exit gracefully.|
-| `addClass(className: string) => void` | Adds a class to the root element. |
-| `removeClass(className: string) => void` | Removes a class from the root element. |
-
-
-#### The full foundation API
-
-##### MDCRadioFoundation.isChecked() => boolean
-
-Returns the value of `adapter.getNativeControl().checked`. Returns `false` if `getNativeControl()`
-does not return an object.
-
-##### MDCRadioFoundation.setChecked(checked: boolean) => void
-
-Sets the value of `adapter.getNativeControl().checked`. Does nothing if `getNativeControl()` does
-not return an object.
-
-##### MDCRadioFoundation.isDisabled() => boolean
-
-Returns the value of `adapter.getNativeControl().disabled`. Returns `false` if `getNativeControl()`
-does not return an object.
+| `getNativeControl() => HTMLInputElement?` | Returns the native radio control |
```

Comment:
Should probably keep ", if available" here?

Reply:
Will there ever be a case when MDC Radio is used without a native control? Or is this just for safety?

Reply:
Realistically, no. I originally wasn't going to comment on this, but then saw a bunch of docs under the foundation that still allude to behavior if the input isn't set.